### ISSUE TITLE
Missing bin path

### ISSUE THREAD

**Issue by Leo6Leo** [Cali0707/git-utils#16]
https://github.com/Cali0707/git-utils/blob/69d2aca4bfaa9041d845bfc2cabb6640a8fc09e6/install.sh#L43

If missing, what should people do next? Maybe provide more context here will be a great thing to have here.

**Comment by Cali0707** [comment]
This is on hold until after #13 is merged
